.jenkins/short-perf-test-cpu.sh

```diff
@@ -7,41 +7,48 @@ cd .jenkins/perf_test
 
 echo "Running CPU perf test for PyTorch..."
 
-# Get last master commit hash
-export PYTORCH_COMMIT_ID=$(git log --format="%H" -n 1)
+pip install awscli
 
-# Get baseline file from https://github.com/yf225/perf-tests
-if [ -f /var/lib/jenkins/host-workspace/perf_test_numbers_cpu.json ]; then
-    cp /var/lib/jenkins/host-workspace/perf_test_numbers_cpu.json perf_test_numbers_cpu.json
-else
-    curl https://raw.githubusercontent.com/yf225/perf-tests/master/perf_test_numbers_cpu.json -O
+export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
```

Comment:
I don't think this is necessary.

Also, point of bash style: put quotes around your variable expansions.